api/tangle.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+import iota
+import numpy as np
+from utils.logging import logging
+
+from config import app
+
+
+def send_to_iota(send_data):
+    # preparing transactions
+    pt = []
+
+    starttime = datetime.now()
+    logging.info("START")
+    pt.append(iota.ProposedTransaction(address=iota.Address(app.config['TARGETADDRESS']),  # 81 trytes long address
+                                       message=iota.TryteString.from_unicode(
+        send_data),
+        # Up to 27 trytes
+        tag=iota.Tag(b'RBTC9D9DCDEAKDCDFD9DSC'),
```

Comment:
May you change it to a configuration parameter?

Reply:
Did you mean `address` and `tag`?

Reply:
Conclusion is no need to change.